perllib/FixMyStreet/Cobrand/Merton.pm

```diff
@@ -38,4 +38,13 @@ sub open311_config {
 
 sub privacy_policy_url { "https://www.merton.gov.uk/legal/privacy-and-cookies" }
 
+sub reopening_disallowed {
+    my ($self, $problem, $user) = @_;
```

Comment:
If my main comment is implemented, you don't need to pass $user in/around, as it will be in `$self->{c}->user`.